api/mysagw/identity/models.py

```diff
@@ -60,9 +60,9 @@ class Identity(UUIDModel, HistoricalModel, TrackingModel):
     SALUTATION_NEUTRAL = "neutral"
 
     SALUTATION_CHOICES = (
-        (SALUTATION_MR, {"de": "Herr", "en": "Mr.", "fr": "Monsieur"}),
-        (SALUTATION_MRS, {"de": "Frau", "en": "Mrs.", "fr": "Madame"}),
-        (SALUTATION_NEUTRAL, {"de": "", "en": "", "fr": ""}),
+        (SALUTATION_MR, SALUTATION_MR),
```

Comment:
If you want translated labels, I think `gettext_lazy` would be the right thing in this context. But I assume you don't actually need/want translated labels here right?

Reply:
I've decided against it, because it's just three values after all and in the backend we need to localize them based on identity.language.